src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    private String name = "Square";
+    private double firstLeg, secondLeg;
```

Comment:
Since both legs of a square are equal, there's no need to have two variables representing them. Use one variable, like 'side', to represent both.